tests/unit/lib/interfaces/qunit.js

```diff
@@ -351,6 +351,18 @@ define([
 			}
 		},
 
+		'extend': {
+			'should have a working expect': function () {
```

Comment:
this test doesn’t test the various `undefined` handling branches so is not complete.